### ISSUE TITLE
Please add crossOrigin settings for QRCode icon

### ISSUE THREAD

**Issue by Kizaruuu** [ant-design/ant-design#48759]
### What problem does this feature solve?

Can't download QRCode with icon

**Comment by kiner-tang** [comment]
It seem that qrcode.react  was fixed this issue last month, I will follow up the modification in antd

**Comment by zhangwei900808** [comment]
I find a case solved this issue. Parsing image to base64 then you can download and work!

```
const [icon, setIcon] = useState('')

async function convertImageToBase64(url) {
        const response = await fetch(url);
        const blob = await response.blob();

        return new Promise((resolve, reject) => {
            const reader = new FileReader();
            reader.onloadend = () => resolve(reader.result);
            reader.onerror = reject;
            reader.readAsDataURL(blob);
        });
    }

    function addIcon() {
        convertImageToBase64('https://gw.alipayobjects.com/zos/rmsportal/KDpgvguMpGfqaHPjicRK.svg').then(base64 => {
            console.log(base64); // 打印Base64字符串
            setIcon(base64)
        })
            .catch(error => {
                console.error('Error:', error);
            });
    }
```

```
<QRCode
    size={size}
    color={color}
    iconSize={size / 4}
    value={text || '-'}
    icon={icon}
    bgColor="#fff"/>

<Button onClick={addIcon}>add icon</Button>
```

**Comment by kiner-tang** [comment]
fixed in #49454
